src/test/node/node-runtime-skipfiles-skip-node-internals.txt

```diff
@@ -0,0 +1,13 @@
+<anonymous> @ ${workspaceFolder}/simpleNode/index.js:1:1
+runInThisContext @ vm.js:122:20 <hidden: Skipped by skipFiles>
+runInThisContext @ vm.js:319:38 <hidden: Skipped by skipFiles>
+Module._compile @ internal/modules/cjs/loader.js:686:28 <hidden: Skipped by skipFiles>
+Module._extensions..js @ internal/modules/cjs/loader.js:734:10 <hidden: Skipped by skipFiles>
+Module.load @ internal/modules/cjs/loader.js:620:32 <hidden: Skipped by skipFiles>
+tryModuleLoad @ internal/modules/cjs/loader.js:560:12 <hidden: Skipped by skipFiles>
+Module._load @ internal/modules/cjs/loader.js:552:3 <hidden: Skipped by skipFiles>
+Module.runMain @ internal/modules/cjs/loader.js:776:12 <hidden: Skipped by skipFiles>
+executeUserCode @ internal/bootstrap/node.js:342:17 <hidden: Skipped by skipFiles>
+startExecution @ internal/bootstrap/node.js:276:5 <hidden: Skipped by skipFiles>
+startup @ internal/bootstrap/node.js:227:5 <hidden: Skipped by skipFiles>
+bootstrapNodeJSCore @ internal/bootstrap/node.js:743:3 <hidden: Skipped by skipFiles>
```

Comment:
I might actually make this assertion a regex, or otherwise test this output manually. The bottom portion of this stacktrace will vary between any different Node.js versions and make this test flake when we upgrade or if someone locally has a slightly different JS version.

Reply:
Done!